pennylane/workflow/execution.py

```diff
@@ -78,7 +78,7 @@
 ]
 
 _mapping_output = (
-    "Numpy",
+    "numpy",

```

Comment:
Potentially we need to keep this `None`.

Reply:
I thought in `_mapping_output`, `numpy` means `numpy`? What's the difference between `numpy` and `None` here?

Reply:
The issue if that the interface gets mapped several times. So we end up with `INTERFACE_MAP[INTERFACE_MAP[None]] == "autograd"`.

Reply:
That's a good point. However, do we have handling for the actual `interface=None`?

Reply:
Considering that in `_mapping_output`, `numpy` was previous called `Numpy`, and we never see a key error raised anywhere, I think it's safe to say that the mapped output is never re-mapped.